### ISSUE TITLE
IIIF Manifests Support/Include Alt Text

### ISSUE THREAD

**Issue by sourslaw** [UMNLibraries/cdm-blacklightify#138]
## Descriptive Summary
- IIIF Manifests include alt text description

## Expected Behavior
- Alt text is included in IIIF manifests . . .

## Notes
- Alt text may be generated through some AI related code (per Theresa's alt text project), some type of integration may have to implemented/supported . . .
- [Alt Text using the 'accessibility' property in HTML annotation bodies](https://github.com/IIIF/api/issues/2319)
- [Consider a provides property on Annotation](https://github.com/IIIF/api/issues/2308)
- [Accessibility Best Practices: IIIF Manifests](https://docs.google.com/document/d/1DFEIfXFZcSV-NgggQpzpe-xuQlxZwg32_FNqID8YQFQ/edit?usp=sharing)
- [Example](https://newspapers.library.wales/view/3414998/3415005/86/)
- [Example](https://theseusviewer.org/?iiif-content=https%3A%2F%2Fupcdn.io%2FkW2K8nf%2Fraw%2Frendering_example.json)
